zfsbootmenu/install-helpers.sh

```diff
@@ -113,12 +113,14 @@ create_zbm_conf() {
 	readonly _ETC_ZFSBOOTMENU_CONF=1
 	EOF
 
+  # shellcheck disable=SC2154
   cat >> "${BUILDROOT}/etc/zfsbootmenu.conf" <<-EOF
 	export HAS_REFRESH="${has_refresh}"
 	export HAS_DISABLED="${has_disabled}"
 	export HAS_BORDER="${has_border}"
 	export HAS_COLUMN="${has_column}"
 	export ZBM_BUILDSTYLE="${ZBM_BUILDSTYLE}"
+	export ZBM_RELEASE_BUILD="${release_build}"
```

Comment:
```suggestion
	export ZBM_RELEASE_BUILD="${zfsbootmenu_release_build}"
```
Then you can just set that argument directly in the modules. Also, maybe normalize the value before writing it, so that `${zfsbootmenu_release_build,,}` maps `y|yes|on|1|t|true` or similar to `yes`.